modules/tensor_mechanics/test/tests/dynamics/time_integration/newmark.i

```diff
@@ -145,48 +138,35 @@
     type = RankTwoAux
     rank_two_tensor = stress
     variable = stress_yy
-    index_i = 0
+    index_i = 1
     index_j = 1
   []
   [strain_yy]
     type = RankTwoAux
     rank_two_tensor = total_strain
     variable = strain_yy
-    index_i = 0
+    index_i = 1
```

Comment:
I think I fixed a test with a similar issue. IIRC it was meant to test along the y direction as is done here.

Reply:
the yy direction? Yeah, I think so. The xy direction is pretty much zero.